src/commands/create.js

```diff
@@ -193,6 +194,25 @@ export default asyncCommand({
 			spinner.succeed('Done!\n');
 		}
 
+		if (argv.less || argv.sass || argv.stylus) {
+			const { less, sass, stylus } = argv;
+			const srcDir = path.resolve(target, 'src');
+
+			let extension;
+
+			if (less) extension = '.less';
+			if (sass) extension = '.scss';
+			if (stylus) extension = '.styl';
+
+			await copy(srcDir, srcDir, {
```

Comment:
I wonder if there is a way to rename here instead of copy+delete?

```js
import glob from 'glob';

let cssFiles = await promisify(glob)('**/*.css', { cwd: srcDir });
let renameToCss = name => fs.rename(name, name.replace(/.css$/, extension));
await Promise.all(cssFiles.map(renameToCss));
```